dashboard/app/models/census/state_cs_offering.rb

```diff
@@ -261,6 +275,15 @@ def self.get_courses(state_code, row_hash)
     when 'IN'
       # A column per CS course with a value of 'Y' if the course is offered.
       IN_COURSE_CODES.select {|course| row_hash[course] == 'Y'}
+    when 'MA'
+      # Don't consider a course as offered at a school if there is no enrollment ("*") or it is not a positive number
+      MA_COURSE_CODES.select do |course|
+        course == row_hash['Course Code'] &&
+        row_hash['Progrmming Included'] == 'Y' &&
```

Comment:
Is the missing "a" deliberate?

Reply:
Yes.  That's how the column is labelled in the spreadsheet provided by the state.